apps/emails-and-messages/src/modules/sendgrid/configuration/sendgrid-metadata-manager.ts

```diff
@@ -1,12 +1,8 @@
+import { GenericConfigurator } from "../../smtp/configuration/smtp-metadata-manager";
```

Comment:
why Sengrid is importing anything from smtp? This is forbidden if we want to have any modularity/boundaries.

GenericConfigurator, if indeed generic, should be moved higher

Reply:
Interface removed since in non-generic service it was not used.